### PR TITLE
Status endpoint

### LINKED ISSUE #888: Add status to endpoints
_NOTE: this is part of the reason the UI fails on the summary page. We used to get a `status` from the `/irs` endpoint but that was removed when we decided that the IRS verification was no longer required. So now we aren't getting a status update beyond `8: validated with errors`._

Currently the following endpoints return a submission status:

- `/institutions/<institutionId>/filings/<period>`
- `/institutions/<institutionId>/filings/<period>/submissions`
- `/institutions/<institutionId>/filings/<period>/submissions/latest`
- `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>`
- `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>/edits/quality`
- `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>/sign`

We also need these endpoints to return the submission status:

- [ ] `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>/edits`
- [ ] `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>/parseErrors`
- [ ] `/institutions/<institutionId>/filings/<period>/submissions/<submissionId>/edits/<type>`
  - including the `macro` and `quality` `POST` responses

If it makes things easier, I don't see the harm in including the `status` on all `/submissions/...` endpoints.

### PR DESCRIPTION
**Pull request by nickgrippin**
Also changes some test data to get filings and submissions from the new panel data.

Closes #888

### CODE REVIEW

**Comment by nickgrippin** [comment]
Side note, this will likely need to change depending on the merge order with #889

File: api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala [hunk 1]
```diff
@@ -55,8 +55,12 @@ trait SubmissionEditPaths
 
             onComplete(fEditSummary) {
               case Success(edits) =>
-                if (format.getOrElse("") == "csv") complete(edits.toCsv)
-                else complete(ToResponseMarshallable(edits))
+                onComplete(getSubmissionStatus(SubmissionId(institutionId, period, seqNr))) {
```
Comment (jmarin) [inline api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala]: Can this nested onComplete be avoided with a `for comprehension`?

File: api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala [hunk 2]
```diff
@@ -138,19 +142,30 @@ trait SubmissionEditPaths
...
-      case Success(edits: MacroResults) =>
-        if (format == "csv") complete("editType, editId\n" + edits.toCsv)
-        else complete(ToResponseMarshallable(edits))
-      case Success(edits: EditResults) =>
-        if (format == "csv") complete("editType, editId, loanId\n" + edits.toCsv(editType))
-        else complete(ToResponseMarshallable(edits))
-      case Success(_) => completeWithInternalError(uri, new IllegalStateException)
+    onComplete(getSubmissionStatus(submissionId)) {
+      case Success(status) =>
+        onComplete(fSingleEdits) {
```
Comment (jmarin) [inline api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionEditPaths.scala]: Same comment as above, can this nested `onComplete` be avoided

**Comment by schbetsy** [comment]
LGTM. @awolfe76 LGTY?

**Pull request merged**